lms/envs/test.py

```diff
@@ -333,7 +333,7 @@
     'TOS': 'tos',
     'HONOR': 'honor',
     'PRIVACY': 'privacy',
-    'JOBS': 'jobs',
+    'CAREERS': 'careers',
```

Comment:
Good catch -- I don't think it really matters since it's just for tests, but consistency certainly doesn't hurt.